hnn_core/network.py

```diff
@@ -438,6 +447,7 @@ def copy(self):
         net_copy = deepcopy(self)
         net_copy.cell_response = CellResponse(times=self.cell_response._times)
         net_copy._reset_drives()
+        net_copy.rec_array = dict()
```

Comment:
Are you sure you want to clear all the arrays? Or just the data in the arrays.

If you do just want to clear them all, I would update the docstring.

Reply:
You're right: I just want to clear the data!